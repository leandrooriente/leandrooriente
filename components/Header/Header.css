.header {
  border: 1px solid #DDD;
}
