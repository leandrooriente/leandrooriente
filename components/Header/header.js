import React from 'react';
import css from './header.css';

const Header = () => (
  <header className={css.header}>
    <h1>OlÃ¡</h1>
  </header>
);

export default Header;
