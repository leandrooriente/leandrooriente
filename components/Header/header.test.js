import React from 'react';
import { mount } from 'enzyme';
import Header from './header';

describe('<Header />', () => {
  it('should render <Header /> component', () => {
    const wrapper = mount(<Header />);

    expect(wrapper).toMatchSnapshot();
  });
});
