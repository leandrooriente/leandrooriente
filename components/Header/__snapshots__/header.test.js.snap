// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render <Header /> component 1`] = `
<Header>
  <header>
    <h1>
      Ol√°
    </h1>
  </header>
</Header>
`;
